lib/cartopy/tests/mpl/test_mpl_integration.py

```diff
@@ -31,13 +31,13 @@
 from cartopy.tests.mpl import MPL_VERSION, ImageTesting
 
 
-_ROB_TOL = 0.5 if ccrs.PROJ4_VERSION < (4, 9) else 0.1
+_ROB_TOL = 0.5 if ccrs.PROJ4_VERSION < (4, 9) else 0.111
 if MPL_VERSION >= '2':
     _STREAMPLOT_IMAGE = 'streamplot'
-elif MPL_VERSION >= '1.5':
```

Comment:
Of course, this doesn't really work properly.
E.G. `(''1.4.3' > '1.17.0')` --> True

Do we have some agreement on this as a temporary kludge ?
Or can we use `distutils.StrictVersion(MPL_VERSION)` instead ?

Reply:
MPL_VERSION is already a `distutil.LooseVersion` will that compare with a tuple?

Reply:
Oh wait, since it's a `LooseVersion` comparing with a string does the correct comparison

Reply:
I thought I'd checked that but you're right -- it's already a LooseVersion, which is fine.
Sorry !

The [other one](https://github.com/SciTools/cartopy/pull/960#pullrequestreview-79148209) does still apply, I think.